pymc3/plots/__init__.py

```diff
@@ -1,4 +1,4 @@
-#   Copyright 2020 The PyMC Developers
+#   Copyright 2021 The PyMC Developers
```

Comment:
I don't think this needs to change (and if it does, I think the change needs doing for the whole codebase?)

Reply:
Good point, let's not change that here then